src/components/SpruceForm/Widgets/LeafyGreenWidgets.tsx

```diff
@@ -263,7 +264,14 @@ export const LeafyGreenRadioBox: React.VFC<
         </RadioBoxLabelContainer>
       )}
       {!!errors?.length && (
```

Comment:
can errors be either undefined or an empty array? I would expect one or the other?